app/scripts/templates/test_template.mustache

```diff
@@ -37,10 +37,12 @@
     <div class="input-row password-row">
       <input type="password" class="password" id="password" pattern=".{8,}" required />
       <input id="show-password" type="checkbox" class="show-password" aria-controls="password" data-novalue />
+      <div class="input-help input-help-focused input-help-signup">{{#t}}Must be at least 8 characters{{/t}}</div>
```

Comment:
Since this is a test template, we can remove the `{{#t}}` and `{{/t}}` from the file.